.storybook/pages/AlongDemo/AlongDemo.module.css

```diff
@@ -10,9 +10,30 @@
 .wireframe-demo__footer {
   padding: 7.5rem 0 2rem;
   display: flex;
-  gap: 3rem;
+  align-items: flex-start;
   flex-wrap: wrap;
   justify-content: center;
+  margin-bottom: 0.5rem;
```

Comment:
This footer divider stuff is mostly copy/pasted from along so it works the same way.